lib/widgets/pages/ride_attendee_scanning_page/scan_results_list.dart

```diff
@@ -54,11 +52,11 @@ class ScanResultsList extends StatelessWidget {
 
 /// This widget represents a single item for the [ScanResultsList].
 class _ScanResultsListItem extends StatelessWidget {
-  const _ScanResultsListItem({required this.device});
+  const _ScanResultsListItem({required this.peripheral});
 
-  final ScannedDevice device;
+  final BluetoothPeripheralWithOwners peripheral;
 
-  Widget _buildMultipleOwnersListItem(BuildContext context, int ownersLength) {
```

Comment:
Minor cleanup on the `switch` below. Now each method returns directly, instead of delegating to a local variable called `child`.